assets/src/blocks/CarouselHeader/SidebarSlide.js

```diff
@@ -24,114 +24,118 @@ export const SidebarSlide = ({
   onDragOverHandler,
   upOrDownHandler,
   goToSlideHandler,
-}) => {
-  const [ isOpened, setIsOpened ] = useState(false);
+} ) => {
+  const [ isOpened, setIsOpened ] = useState( false );
 
-  const onFocalPointsChange = (index, value) => {
+  // Changed index to idx to avoid eslint errors.
+  const onFocalPointsChange = ( idx, value ) => {
```

Comment:
Here instead of renaming to `idx` couldn't we just remove that parameter? If it's always gonna be the `index` prop anyway? 🤔